src/linker/Linker.Steps/DiscoverCustomOperatorsHandler.cs

```diff
@@ -0,0 +1,180 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Mono.Cecil;
+
+namespace Mono.Linker.Steps
+{
+	public class DiscoverOperatorsHandler : IMarkHandler
+	{
+		LinkContext _context;
```

Comment:
a few could be reasonly

Reply:
_context can't be since it's not passed to the ctor, but I made the HashSet readonly. I changed the Dictionary to be lazily initialized.